src/stateManager/utils/FraudProofService.ts

```diff
@@ -8,11 +8,15 @@ import {
     WrongGenesisProofStruct
 } from "@typechain-types/contracts/V1/types/FraudProofTypes";
 import { FraudProofStruct } from "@typechain-types/contracts/V1/types/ProofTypes";
-import { ZeroHash } from "ethers";
 import { Address, Hash } from "@/types/types";
 import { Codec, FraudStruct } from "@/utils/Codec";
 import { FraudProofType, toSolidityFraudProofType } from "@/types/sol-enums";
 
+const createEmptySignedBlock = (): SignedBlockStruct => ({
+    encodedBlock: "0x",
```

Comment:
use ethers zeroHash

Reply:
yeah I'm not sure about this one - '0x' means it's empty, `zeroHash` has the same dataSize/length as a hash, but it's filled with `0s` - what I think would be correct here is to fill the data structure with 0s but in a representation where it would still decode correctly to the underlying structure - not sure will this be a problem - tests are needed